2024-04-13 19:18:15 ERROR Elements not present after 0.5 seconds on page: https://capital.com/trading/platform/
2024-04-13 19:18:15 ERROR 
Traceback (most recent call last):
  File "/home/runner/work/TestCapitalComPySe/TestCapitalComPySe/pages/base_page.py", line 85, in inner_function
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/home/runner/work/TestCapitalComPySe/TestCapitalComPySe/pages/base_page.py", line 304, in element_is_visible
    return Wait(self.driver, timeout).until(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/runner/.cache/pypoetry/virtualenvs/testcapitalcompyse-CcJ6hTD8-py3.12/lib/python3.12/site-packages/selenium/webdriver/support/wait.py", line 105, in until
    raise TimeoutException(message, screen, stacktrace)
selenium.common.exceptions.TimeoutException: Message: 
Stacktrace:
#0 0x55f34c8fb863 <unknown>
#1 0x55f34c5f18c6 <unknown>
#2 0x55f34c63c618 <unknown>
#3 0x55f34c63c6d1 <unknown>
#4 0x55f34c67f744 <unknown>
#5 0x55f34c65e5cd <unknown>
#6 0x55f34c67cc19 <unknown>
#7 0x55f34c65e343 <unknown>
#8 0x55f34c62f593 <unknown>
#9 0x55f34c62ff5e <unknown>
#10 0x55f34c8bf84b <unknown>
#11 0x55f34c8c37a5 <unknown>
#12 0x55f34c8ad571 <unknown>
#13 0x55f34c8c4332 <unknown>
#14 0x55f34c89287f <unknown>
#15 0x55f34c8ea728 <unknown>
#16 0x55f34c8ea8fb <unknown>
#17 0x55f34c8fa9b4 <unknown>
#18 0x7fe0a4c94ac3 <unknown>